src/upload.ts

```diff
@@ -856,14 +855,10 @@ async function login(localPage: Page, credentials: Credentials, messageTransport
             smsAuthSelector
         )
         if (isOnSmsAuthPage) {
-            const rl = readline.createInterface({
-                input: process.stdin,
-                output: process.stdout
-            });
-            let code = await rl.question('Enter the code that was sent to you via SMS: ');
+            if (!messageTransport.prompt) throw new Error('Prompt Method Required to Receive SMS')
+            let code = await messageTransport.prompt('Enter the code that was sent to you via SMS: ')
```

Comment:
I think we should handle no code being inputed, either through a try catch, or by allowing the signature to return null (so then we can check if "code" is empty or not).
We probably want to close the browser and end end the execution of the function if the user doesn't input a code.

Reply:
Gonna add the input and exception handling.